torchci/lib/drciUtils.ts

```diff
@@ -24,14 +29,73 @@ Note: Links to docs will display an error until the docs builds have been comple
 }
 
 export function formDrciComment(
-    pr_num: number,
-    pr_results: string = "",
+  pr_num: number,
+  pr_results: string = "",
+  sevs: string = ""
 ): string {
-    const header = formDrciHeader(pr_num);
-    const comment =
-        `${DRCI_COMMENT_START}
+  const header = formDrciHeader(pr_num);
+  const comment = `${DRCI_COMMENT_START}
 ${header}
+${sevs}
 ${pr_results}
 ${DRCI_COMMENT_END}`;
-    return comment;
+  return comment;
+}
+
+
+export async function getDrciComment(
+  octokit: Octokit,
+  owner: string,
+  repo: string,
+  prNum: number
+): Promise<{ id: number; body: string }> {
+  const commentsRes = await octokit.rest.issues.listComments({
+    owner: owner,
+    repo: repo,
+    issue_number: prNum,
+  });
+  for (const comment of commentsRes.data) {
+    if (comment.body!.includes(DRCI_COMMENT_START)) {
+      return { id: comment.id, body: comment.body! };
+    }
+  }
+  return { id: 0, body: "" };
+}
+
+export function getActiveSEVs(issues: IssueData[]): string {
+  const activeSEVs = issues.filter(
+    (issue: IssueData) => issue.state === "open"
+  );
+  if (activeSEVs.length === 0) {
+    return "";
+  }
+  const [merge_blocking, not_merge_blocking] = _.partition(
```

Comment:
It would be nicer IMO to have this function `getActiveSEVs` returns the lists of merge blocking and non merge blocking sevs instead of a formatted string.  The reason is that I think this function can then be reusable elsewhere.  You can move the string formatting part to a new function, i.e. `formDrciSevBody` or something.  Thoughts?

Reply:
doen